pages/linux/ipset.md

```diff
@@ -0,0 +1,24 @@
+# ipset
+
+> A quick and easy tool to create IP sets for firewall rules
+
```

Comment:
We generally tend to have a `More information: <hyperlink>` section as well if possible.

Reply:
Please give me an example!

Reply:
sure have a look at this: https://github.com/tldr-pages/tldr/blob/main/pages/common/fzf.md

Reply:
When using man pages we prefer `manned.org`, so that would be <https://manned.org/ipset>.

Reply:
```suggestion
> More information <https://manned.org/ipset>.
```